.travis.yml

```diff
@@ -2,14 +2,15 @@ language: go
 sudo: false
 
 go:
-  - master
-  - "1.10"
-  - "1.11"
+  - "1.12"
```

Comment:
https://docs.travis-ci.com/user/languages/go/#specifying-a-go-version-to-use
Recommend to use `"1.12.x"` to test against the latest `1.12` version